addons/base_automation/tests/test_mail_composer.py

```diff
@@ -34,11 +34,12 @@ def test_mail_composer_test_tour(self):
             'model_id': self.env.ref('mail.model_mail_compose_message').id,
         })
         automation.write({'action_server_ids': [(4, server_action.id)]})
-
+        user = self.env["res.users"].create({"name": "Not A Demo User", "login": "nadu"})
         self.start_tour(
             f"/web#id={self.user_employee.partner_id.id}&model=res.partner",
             "mail/static/tests/tours/mail_composer_test_tour.js",
             login=self.user_employee.login
         )
-
+        message = self.env["mail.message"].search([], order="id desc")
```

Comment:
Can't we use existing mocks and be sure we produced a message during the tour, and check its values ?

Reply:
I don't know about them but that looks like a good idea.